### PR TITLE
 Harmonization of case report spacing for some lists and #panel_tabels

### PR DESCRIPTION
**Pull request by TereseBo**
This PR adds some space between <li> in some tables as well as replaces some <td> width attributes (deprecated in html5) with corresponding inline styles. Some <br> have been moved as to not render twice for the last child in a card as well as for unifying the structure of variant_content for different variant types. The aim for the change was to unify the visual expression of the nested tables in the case_report cards and to improve readability.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. Go to /case_report for some cases and verify the styling is coherent.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [ ] code approved by
- [x] tests executed by TB

### CODE REVIEW

**Comment by TereseBo** [comment]
On main branch the second #panel-table in a variant is either missing a width or has one specified with a faulty syntax resulting in variable widths of the table:
image
Screenshot 2024-04-10 at 12 45 21

On this branch the syntax has been corrected, added to both instances of the table and also somewhat increased.
image

image


On main branch, when no phenotypes or diagnoses have been specified for the case, text is rendered after a break:
image

On this branch the spacing is changed:
image


On main branch the card for pinned variants renders an additional <br> compared to other cards when empty due to the pinned variants being present in causatives:
image

On this branch this break has been moved:
Screenshot 2024-04-10 at 12 54 56

On main branch the transcript list does not have space between the list items:
image

On this branch spacing has been added:
Screenshot 2024-04-10 at 12 57 48

File: scout/server/blueprints/cases/templates/cases/case_report.html [hunk 1]
```diff
@@ -551,7 +558,7 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
       </table>
       <table class="table table-sm">
         <tr>
-          <td style="width=45%">
```
Comment (northwestwitch) [inline scout/server/blueprints/cases/templates/cases/case_report.html:554]: Ah OK, didn't know that you needed ; also here! 💯

**Review by northwestwitch — APPROVED** [review]
Looks good! 🚀

File: scout/server/blueprints/cases/templates/cases/case_report.html (hunk 1 above)
Comment (TereseBo) [reply scout/server/blueprints/cases/templates/cases/case_report.html:554]: Yes, it's also : instead of = inside of the css

**Pull request merged**